### PR TITLE
add common schema for firepit

### PR DESCRIPTION
**Pull request by charliewutw1**
This PR contains to create a common schema for two functions  `match` and `in_subnet`

### CODE REVIEW

File: firepit/pgstorage.py [hunk 1]
```diff
@@ -149,18 +151,25 @@ def __init__(self, dbname, url, session_id=None):
 
         logger.debug("Connection to PostgreSQL DB %s successful", dbname)
 
+    def _create_firepit_common_schema(self):
+        try:
+            self._execute(f'CREATE SCHEMA IF NOT EXISTS "firepit_common";')
```
Comment (pcoccoli) [inline firepit/pgstorage.py]: You don't need an f-string here.
If the schema *does* exist, will this raise an exception?  We only need to create this once, so we need a way to check if the schema already exists and don't bother creating it if it does.

Comment (pcoccoli) [reply firepit/pgstorage.py]: Something like the `SELECT (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES...` that we do for the session schema to decide if we should call `_setup()`.

**Review by pcoccoli — CHANGES_REQUESTED** [review]

**Comment by charliewutw1** [comment]
attached unit test report against local postgres

```

$ FIREPITDB="postgresql://charlie:c1234@localhost:5432/stixdata" pytest
=============================================================================================== test session starts ================================================================================================                                                                                                                                                                                             

tests/test_custom_objects.py .                                                                                                                                                                               [  0%]
tests/test_errors.py ..................                                                                                                                                                                      [ 13%]
tests/test_props.py ..............................                                                                                                                                                           [ 36%]
tests/test_query.py .......................................                                                                                                                                                  [ 64%]
tests/test_storage.py ..................................... 
tests/test_validate.py ........... 
   ```

**Review by pcoccoli — APPROVED** [review]
Looks good

**Pull request merged**